mongodbatlas/fw_resource_mongodbatlas_alert_configuration.go

```diff
@@ -671,6 +677,7 @@ func newTFNotificationModelList(matlasSlice []matlas.Notification, currStateNoti
 				TeamID:         conversion.StringNullIfEmpty(value.TeamID),
 				TypeName:       conversion.StringNullIfEmpty(value.TypeName),
 				Username:       conversion.StringNullIfEmpty(value.Username),
+				NotifierID:     types.StringValue(value.NotifierID),
```

Comment:
Is `NotifierID` in the manual SDK? Do you plan to migrate the resource to the new SDK?

Reply:
oh I just noticed the other PR. You can use the manual go SDK of your PR here by running 
`go get go.mongodb.org/atlas@INTMDB-981`

Reply:
Thanks for the tip, pushed the temporary change. Using the manual SDK as I already had the changes available in a branch.